### PR TITLE
Created pages and navigation

### PR DESCRIPTION
**Pull request by anastasiyaShcherenkova**
Created the main components of the page: main, nav, header. Added page components and navigation.

### CODE REVIEW

**Review by csi-sergeykhaustovich — COMMENTED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -26,22 +26,21 @@ You are about to create weather application with location detection.
...
 4. Info page where user can see some information about the service.
 5. Feedback page with a form for feedback on your site: please, create the form in survey style with some simple questions.
    Implement form submitting mechanism and save it in localStorage. For now don't bother yourself saving form data in DB.
 6. Implement Dark/Light mode for your app. There should be some sort of switcher that changes current view.
 
-
 ### P.S.
 
-There is a branch <name>_<surname> created for you in original repository. Your task is to fork from this repo and work 
+There is a branch <name>\_<surname> created for you in original repository. Your task is to fork from this repo and work
```
Comment (JuliaShavyrova) [inline README.md:45]: are you sure this file should be changed?

File: src/App.jsx [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+
+import { BrowserRouter, Route } from 'react-router-dom';
+
 function App() {
-  return <div>Hello world!</div>;
+  return (
+    <BrowserRouter>
+      <div className={`${classes.container} ${classes.container_vertical}`}>
+        <Header />
+        <div className={`${classes.container} ${classes.container_horizontal}`}>
```
Comment (JuliaShavyrova) [inline src/App.jsx]: if container_horizontal and container_vertical are not common classes that can be used by every component, their names should be more specific

File: src/components/main/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import classes from './main.scss';
+
+import Routes from '../../config/Routes';
+
+function Main() {
```
Comment (JuliaShavyrova) [inline src/components/main/Main.jsx]: please rename components according to our last lecture content

File: src/config/Routes.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            <Route 
+                path='/worldWeather'
+                exact
+                component={WorldWeather}
+            />
+            <Route 
+                path='/contactUs'
+                exact
+                component={Contacts}
+            /> */}
```
Comment (JuliaShavyrova) [inline src/config/Routes.jsx]: please remove unused code

**Comment by anastasiyaShcherenkova** [comment]
@JuliaShavyrova, I made some adjustments.

**Pull request merged**